app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -32,6 +32,9 @@ def edit
       if @records.all? { |intake| intake.hashed_ssn.nil? }
         sign_in_and_redirect
       end
+      if @records.any?
+        @records.each { |record| record.reset_failed_attempts_if_unlocked! }
+      end
```

Comment:
both login methods (via `VerificationCodeController` and `IntakeLoginsController`) eventually feed into this `#edit` action, so seems apt to reset the failed attempts when we get to this action